channels.yaml

```diff
@@ -268,3 +282,18 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  #  v1.24.5+k3s1 was never released
+  - version: v1.24.6-rc1+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v3
+    agentArgs: *agentArgs-v2
+    featureVersions: *featureVersions-v1
+  #  v1.25.0+k3s1 was never released through KDM
+  #  v1.25.1+k3s1 was never released
+  - version: v1.25.2-rc1+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
```

Comment:
v1.25 is supported on v2.6.7. The current milestone set in rancher/rancher for that is v2.7.1. I think for this v2.6 line we should just remove it.